components/persistor/lib/schema.ts

```diff
@@ -137,8 +143,23 @@ module.exports = function (PersistObjectTemplate) {
                 schema.indexes = schema.indexes || [];
                 var keyName = 'idx_' + schema.table + '_fk_' + val.id;
                 if (isOTM && !indexes[keyName]) {
-                    indexes[keyName] = true;
                     schema.indexes.push({name: keyName, def: {columns: [val.id], type: 'index'}});
+                    indexes[keyName] = true;
+                }
+            }
+
+            // make sure that we also add FK indexes for one to one object reference relationships
+            function addOneToOneIndexes(referencedObjectFromSchema) {
+                schema.indexes = schema.indexes || [];
+                const keyName = 'idx_' + schema.table + '_fk_' + referencedObjectFromSchema.id;
```

Comment:
can you make this a template string?

Reply:
hey! to my eyes with all those underscores it's actually harder to read that line all mushed together with the template string syntax (I tried implementing this) - I think the only real benefit of template literals here would be readability since it's not a multi-line statement or in need of escaping quote characters, so if it's not improving readability probably just gonna keep as is.